instrumentation-core/spring/build.gradle

```
@@ -0,0 +1,36 @@
+plugins {
+	id 'org.springframework.boot' version '2.3.0.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.9.RELEASE'
+	id 'java'
+}
+
+group = 'io.opentelemetry.contrib'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
```

I'd suggest supporting Java 8+, unless there's a specific reason to only support Java 11+
```suggestion
sourceCompatibility = '8'
```